nano/lib/blocks.cpp

```diff
@@ -1866,6 +1866,11 @@ bool nano::block_sideband::deserialize (nano::stream & stream_a, nano::block_typ
 	return result;
 }
 
+std::string nano::block_sideband::to_json () const
```

Comment:
Thank you for the effort, but I would expect to see something much more elegant and much easier to read.